nanoFramework.Device.Bluetooth/BluetoothEventType.cs

```diff
@@ -100,7 +102,41 @@ public enum BluetoothEventType
         /// <summary>
         /// Fired when a value on connected device has changed, notify.
         /// </summary>
-        AttributeValueChanged
+        AttributeValueChanged,
+
+        // Add further central events here >>>>>>
+
 
+        // ====  Session / Pairing / Security group events ====
+
+        /// <summary>
+        /// Event when client connects
+        /// </summary>
+        ClientConnected,
+
+        /// <summary>
+        /// Event when client disconnects
+        /// </summary>
+        ClientDisconnected,
+
+        /// <summary>
+        /// Event when session details updated
+        /// </summary>
+        ClientSessionChanged,
+
+        /// <summary>
+        /// Fired when a passkey action is requested 
+        /// </summary>
+        PassKeyActions,
+
+        /// <summary>
+        /// Fired when a passkey action for numeric comparsion is requested 
+        /// </summary>
+        PassKeyActions_numcmp,
```

Comment:
Better use full name to stick with our convensions

```suggestion
        PassKeyActionsNumericComparison,
```